feinstaub/nginx.conf.sigil

```diff
@@ -11,9 +11,10 @@ server {
   {{ if $.NOSSL_SERVER_NAME }}server_name {{ $.NOSSL_SERVER_NAME }}; {{ end }}
   access_log  /var/log/nginx/{{ $.APP }}-access.log;
   error_log   /var/log/nginx/{{ $.APP }}-error.log;
-{{ if (and (eq $listen_port "80") ($.SSL_INUSE)) }}
-  return 301 https://$host:{{ $.NGINX_SSL_PORT }}$request_uri;
-{{ else }}
```

Comment:
This redirect is for when SSL is used. Are we removing it coz we want to allow insecure connection?

Reply:
@karimkawambwa These are changes you've made.. I'd think you'd be able to answer questions on them?

Reply:
@DavidLemayian apologies for the lack of explanation. So this change was originally already running in production but it was untracked. What am doing here is adding the untracked change. It seems from the error logs, the sensors are not handling redirects well. When the 301 redirect happens, the methods are change from POST to GET therefore the sensors fail to send data. I think this untracked change addressed that. I have requested Taha for a sensor so that we can see how we can start using secure connection. But for now 301 redirect will change the request method and the other options 307/308 did not work either when I tried them.

Reply:
@chegejames confirmed the GSM-based sensors can't be used w/ SSL. Also as far as he's concerned, the cost of doing HTTPS (packet size) seems a bit too high vs benefit.